src/App.js

```diff
@@ -1,4 +1,4 @@
-import "./App.css";
+import "./styles.App.scss";
```

Comment:
Should this be './styles/App.scss'? Also, are we using App.js for anything right now? As far as I recall, we moved everything to the Root.jsx (and index.js) isntead